src/views/pages/groups/components/Group1.js

```diff
@@ -0,0 +1,99 @@
+import PropTypes from "prop-types";
+
+// material-ui
+import { styled, useTheme } from "@mui/material/styles";
+import { Avatar, Box, List, ListItem, ListItemAvatar, ListItemText, Typography } from "@mui/material";
+
+// project imports
+import MainCard from "ui-component/cards/MainCard";
+import TotalIncomeCard from "ui-component/cards/Skeleton/TotalIncomeCard";
+
+// assets
+import TableChartOutlinedIcon from "@mui/icons-material/TableChartOutlined";
+
+// styles
+const CardWrapper = styled(MainCard)(({ theme }) => ({
+    backgroundColor: theme.palette.primary.dark,
+    color: theme.palette.primary.light,
+    overflow: "hidden",
+    position: "relative",
+    "&:after": {
+        content: '""',
+        position: "absolute",
+        width: 210,
+        height: 210,
+        background: `linear-gradient(210.04deg, ${theme.palette.primary[200]} -50.94%, rgba(144, 202, 249, 0) 83.49%)`,
+        borderRadius: "50%",
+        top: -30,
+        right: -180
+    },
+    "&:before": {
+        content: '""',
+        position: "absolute",
+        width: 210,
+        height: 210,
+        background: `linear-gradient(140.9deg, ${theme.palette.primary[200]} -14.02%, rgba(144, 202, 249, 0) 77.58%)`,
+        borderRadius: "50%",
+        top: -160,
+        right: -130
+    }
+}));
+
+// ==============================|| DASHBOARD - TOTAL INCOME DARK CARD ||============================== //
+
+const Group1 = ({ isLoading }) => {
```

Comment:
Changes seem unrelated to FE-12, please move to correct Issue Branch if they're out of scope for this issue